src/locale/languages/de.js

```diff
@@ -1,7 +1,43 @@
-// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
-// Link: https://github.com/gitpoint/git-point/pull/260
-
-export const de = {
+module.exports = {
+  '{actor} commented on commit': '{actor} kommentierte den commit',
+  '{actor} created branch {ref} at {repo}':
+    '{actor} hat Branch erstellt {ref} bei {repo}',
+  '{actor} created tag {ref} at {repo}':
+    '{actor} hat Tag erstellt {ref} bei {repo}',
+  '{actor} created repository {repo}': '{actor} hat Repository erstellt {repo}',
+  '{actor} deleted branch {ref} at {repo}':
+    '{actor} hat Branch gelöscht {ref} bei {repo}',
+  '{actor} deleted tag {ref} at {repo}':
+    '{actor} hat Tag gelöscht {ref} bei {repo}',
+  '{actor} forked {repo} at {fork}': '{actor} forked {repo} bei {fork}',
+  '{actor} created the {repo} wiki': '',
+  '{actor} edited the {repo} wiki': '',
+  '{actor} commented on pull request {issue} at {repo}':
+    '{actor} kommentierte on Pull Request {issue} bei {repo}',
+  '{actor} commented on issue {issue} at {repo}':
+    '{actor} kommentierte on Issue {issue} bei {repo}',
+  '{actor} opened issue {issue} at {repo}':
+    '{actor} eröffnet Issue {issue} bei {repo}',
+  '{actor} reopened issue {issue} at {repo}':
+    '{actor} erneut geöffnet Issue {issue} bei {repo}',
+  '{actor} closed issue {issue} at {repo}':
+    '{actor} geschlossen Issue {issue} bei {repo}',
+  '{actor} edited {member} at {repo}': '{actor} bearbeitet {member} bei {repo}',
+  '{actor} removed {member} at {repo}': '{actor} löschte {member} bei {repo}',
+  '{actor} added {member} at {repo}': '{actor} hinzugefügt {member} bei {repo}',
+  '{actor} made {repo} public': '{actor} hat {repo} veröffentlicht',
+  '{actor} opened pull request {pr} at {repo}':
+    '{actor} eröffnet Pull Request {pr} bei {repo}',
+  '{actor} reopened pull request {pr} at {repo}':
+    '{actor} erneut geöffnet Pull Request {pr} bei {repo}',
+  '{actor} merged pull request {pr} at {repo}': '',
+  '{actor} closed pull request {pr} at {repo}':
+    '{actor} geschlossen Pull Request {pr} bei {repo}',
+  '{actor} commented on pull request {pr} at {repo}': '',
+  '{actor} pushed to {ref} at {repo}': '',
+  '{actor} published release {id}': '{actor} veröffentlicht Release {id}',
+  '{actor} starred {repo}': '',
```

Comment:
reconstructed all translations for the events screen from the old translations bits

I didn't remove the old translations as they might be used somewhere else. We will nuke all old translations once the transition is completed.